### PR TITLE
Akka integration

### PR DESCRIPTION
**Pull request by chef-plankton**
[ ] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-frontend/blob/master/CONTRIBUTING.md) first
[ ] If your PR is work in progress, open it as `draft`
[ ] Before requesting a review, all the checks need to pass
[ ] Explain what your PR does

### CODE REVIEW

File: apps/web/.env.production
Comment (MasterGordon) [inline apps/web/.env.production:1]: What's the intend behind this change?
We already have sentry integrated from our end. So there is no need for additional setup

File: apps/web/src/utils/sentry.ts
Comment (MasterGordon) [inline apps/web/src/utils/sentry.ts:1]: Weird changes

File: apps/web/.env.production
Comment (chef-plankton) [reply apps/web/.env.production:1]: Hi
Last time I asked about Sentry in group, Simon told me that you are not using it. if you are using it we can just change env variables and you can give us access to sentry. if else we can config new sentry for us. but we need to track our errors. any suggestion?

Comment (MasterGordon) [reply apps/web/.env.production:1]: Yeah we have started using sentry ourselves a few months ago. I will talk with Simon about this. But I really like the idea of utilizing it for this logging.

Comment (chef-plankton) [reply apps/web/.env.production:1]: Ok thanks.
Please tell us the results

**Pull request merged**